kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/BaseGroup.java

```diff
@@ -413,16 +413,16 @@ public void addMember(String userId, String roleId, boolean active, boolean prov
 	{
 		try {
 			insertMember(userId, roleId, active, provided);
-		} catch (IllegalStateException ise) {
-			log.warn(ise.getMessage());
+		} catch (IllegalStateException | AuthzRealmLockException iseorarle) {
+			log.warn(iseorarle.getMessage());
```

Comment:
```suggestion
		} catch (AuthzRealmLockException arle) {
			log.warn(arle.toString());
```

Reply:
What's the benefit of catching them separately?